dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/controller/impl/WorkUnitThreadControllerImpl.java

```diff
@@ -122,6 +133,27 @@ private boolean startWorkUnitIfNotStopping() {
                                                                                              runtimeWorkUnitExecution.getTopLevelInstanceId(),
                                                                                              runtimeWorkUnitExecution.getTopLevelExecutionId() },
                                                    logger);
+
+                // If we're not a split/flow we're a job, so log the original JSL
+                if (!(runtimeWorkUnitExecution instanceof RuntimeSplitFlowExecution)) {
```

Comment:
At one time, the thought was that the flow through this class was clearer by pulling out the job/partition/split-flow specific stuff into the AbstractControllerHelper....e.g. make a new AbstractControllerHelper method for this new divergence.     You could argue at the abstract method level it's actually more confusing...but..just noting this was the idea of the existing organization.

Reply:
ok